backend/common/rhnConfig.py

```diff
@@ -450,7 +450,7 @@ def read_file(filename):
     reads a text config file and returns its lines in a list
     """
     try:
-        lines = open(filename, 'rb').readlines()
+        lines = open(filename, 'r').readlines()
```

Comment:
Reason: avoid `TypeError: a bytes-like object is required, not 'str'` at line 458 (below).

https://stackoverflow.com/a/33054552/4055246

Reply:
I rolled back this, as then `read_file` works, and the parser as well... but nothing is parsed.

Reply:
Still not clear to me why, if this function is just reading a text config file and return its lines in a list, it's doing so in a so strange way.

Reply:
Rewritten to read as plain text, as python3 is already handling all strings as UTF-8.